gps/app/locales/it.json

```diff
@@ -60,7 +69,7 @@
     "choose_provider_subtitle": "Per essere informato sulle possibilità di esposizione devi aggiungere almeno una Istituzione Sanitaria.",
     "choose_provider_title": "Seleziona le Istituzioni Sanitarie",
     "commitment": "Missione",
-    "commitment_para": "Safe Paths registra i dati in piena sicurezza e controlla l'interazione con altre persone usando la tua posizione.\nI tuoi dati non lasceranno MAI il tuo telefono senza il tuo consenso",
+    "commitment_para": "Safe Paths è un progetto internazionale per sviluppare strumenti gratuiti, di pubblico dominio e attenti alla privacy per i cittadini, funzionari della sanità pubblica e comunità al fine di mitigare la curva dei contagi di COVID-19, ridurre la paura e prevenire uno stato di sorveglianza come risposta alla pandemia.\n\nPer ulteriori informazioni, visitare il sito",
```

Comment:
this is a really long line and the line break comes much later, does it look okay on the emulator?

Reply:
Looks fine:

<img width="322" alt="Screen Shot 2020-05-18 at 5 41 47 PM" src="https://user-images.githubusercontent.com/702990/82272249-db98f000-992e-11ea-8106-0f5369daa1ae.png">